src/database/schemamanager.h

```diff
@@ -26,8 +29,15 @@ class SchemaManager {
         return m_currentVersion;
     }
 
+    /// If the current version has no backwards compatibility info, it is
```

Comment:
Please omit the implementation details: "Checks if the current schema version is backwards compatible with the given targetVersion."

Reply:
But this "implementation detail" is extremely important, especially if you write a new schema revision. I wouldn't consider this a "detail", rather an essential information about the logic.